src/plugins/vis_types/timelion/server/series_functions/points.test.js

```diff
@@ -63,7 +62,7 @@ describe('points.js', () => {
       return invoke(fn, [seriesList, null, null, null, null, 'beer'])
         .then(expect.fail)
         .catch((e) => {
-          expect(e).to.be.an('error');
+          expect(e).to.be.a(Error);
```

Comment:
Pretty sure `a()` and `an()` are the same function in `@kbn/expect`, why change it?